tests/integration/test_rerelate.py

```diff
@@ -42,7 +42,7 @@ async def deploy(self, ops_test: OpsTest):
 
 related_apps = [
     RelatedApp("alertmanager", "ch:alertmanager-k8s", "alertmanager", {}),
-    RelatedApp("cassandra", "ch:cassandra-k8s", "logging", {"heap_size": "1g"}),
+    RelatedApp("grafana-agent", "ch:grafana-agent-k8s", "logging", {}),
 ]
```

Comment:
Add traefik to this list?

Reply:
I added a new specific itest for traefik